templates/zerver/emails/custom_email_base.pre.html

```diff
@@ -14,6 +14,7 @@
 {% block manage_preferences %}
     {% if remote_server_email %}
     <p>You are receiving this email to update you about important changes to Zulip's Terms of Service.</p>
+    <a href="{{ unsubscribe_link }}">{{ _("Unsubscribe from marketing emails") }}</a>
```

Comment:
The "from marketing emails" language is inaccurate for this case. The actual situation in `send_custom_server_email` is that this is an option for deactivating your Zulip server registration, which is in line with the "changes in Terms of Service" description above -- this template block is for a class of mandatory emails. Probably we'll want to change the rest of the copy in this block if we use this code path again... so maybe it's fine to write it this way; we only ever us this code path manually, and I don't really want to add an extra translated string that's possibly unused.

```
        context = {                                                                                     
            "remote_server_email": True,                                                                
            "hostname": server.hostname,                                                                
            "unsubscribe_link": generate_confirmation_link_for_server_deactivation(                     
                server, 60 * 24 * 2                                                                     
            ),                                                                                          
        }                                                                                               
```

Reply:
```diff
- <a href="{{ unsubscribe_link }}">{{ _("Unsubscribe from marketing emails") }}</a>
+ <a href="{{ unsubscribe_link }}">Unsubscribe</a>
```

I think it's best to remove the translation tag because the bodies of such emails are not translated. Also, the "Zulip's Terms of Service" line just above is also not translated so it's better to not translate this at all ?